lib/RestructureCFG/ScopeGraphUtils.cpp

```diff
@@ -166,6 +166,7 @@ BasicBlock *ScopeGraphBuilder::makeGotoEdge(BasicBlock *Source,
                                              F);
   IRBuilder<> Builder(Context);
   Builder.SetInsertPoint(GotoBlock);
+  Builder.SetCurrentDebugLocation(Source->getTerminator()->getDebugLoc());
```

Comment:
@andrealinux1 is taking care of this in a PR of its own. Anyway this is fine. Andrea is just going to add a few others in his PR that are currently not in `develop` and that you couldn't have catched.

Reply:
@pfez ping

Reply:
Yes, I already have another PR from @andrealinux1 that does all this and some more.